doc/neps/nep-0036-fair-play.rst

```diff
@@ -0,0 +1,177 @@
+of behavior the community would like to see, and which we consider
+troublesome, bothersome, and unacceptable.
+
+Motivation
+----------
+
+It has come to our attention that modified versions of NumPy are
+distributed by external entities.  Sometimes, the patches are
+innocuous, such as when they allow NumPy to function on the target
+system.  In other cases, they add new functionality which causes
+problems for the community.
+
+For example, in December 2018, through the filing of `Issue #12515
+<https://github.com/numpy/numpy/issues/12515>`__, it became known to
+the NumPy project that an external company had published a version of
+NumPy with an extended footprint.  In this specific instance, a bug
+was filed against the `np.erf` function---a function that NumPy itself
+had never provided.
+
+In the months that followed, several conversations was had with the
+entity involved, and a better understanding emerged of needs on either
+side.  The entity agreed to publish the patches made against Python,
+and a special label was added on the NumPy issue tracker to identify
+issues pertaining to this specific distribution.
+
+In another example, a security issue (CVE-2019-6446) was filed against
+NumPy.  This forced many distributions to take action, most often by
+changing a default keyword value, unfortunately again leading to
+multiple versions of the NumPy API being deployed, varying across
+distribution.
+
+These are only two of many examples of such modified distributions,
+which cause wasted time for our developers: identifying the release
+from which the error came, how the release was modified from the
+version we provide, and whether we are able to address related issues
+(if not, where to relay the report to).
+
+During a community call on XX/XX/XX the community decided to draft a
+set of guidelines for companies and external developers to use when
+they modify and redistribute NumPy.
+
+Scope
+-----
+
+This document aims to define a minimal set of rules that, when
+followed, will placate the NumPy developers.  Our hope, however, is
+that developers who think that they need to modify NumPy will consider
+contributing to the project, or using one of the many mechanisms we
+have for allowing our APIs to operate on externally defined objects.
+
+We encourage and exhort developers to first discuss their problem with
+the NumPy developers on the `mailing list <>`__ before resorting to
+patching and releasing their own version of the library.
+
+Fair Play Rules
+---------------
+
+1. Do not reuse the NumPy name for projects not developed by the NumPy
+   community.
+
+   At time of writing, there are only a handful of `numpy`-named
+   packages developed by the community, including `numpy`,
+   `numpy-financial`, and `unumpy`.  We ask that external packages not
+   include the phrase `numpy`, i.e., avoid names such as
+   `mycompany_numpy`.
+
+   NumPy is a trademark owned by NumFOCUS.
```

Comment:
which means using these names probably constitutes a trademark violation in the US, right?

Reply:
> which means using these names probably constitutes a trademark violation in the US, right?

Good point, but I'm not sure adding stronger language would necessarily be an improvement. E.g. something like:

 > NumPy is a trademark owned by NumFOCUS. Unauthorized use of the NumPy name or logo may constitute a violation of this trademark.

but I'd be hesitant to add wording that invites legal discussions in the NEP itself.

Reply:
I am following up on the NumPy trademark.